SYCL/Basic/group_async_copy.cpp

```diff
@@ -153,6 +197,71 @@ int main() {
   for (int Stride = 1; Stride < WorkGroupSize; Stride++) {
     if (test<int>(Stride))
       return 1;
+    if (test<int2>(Stride))
+      return 1;
+    if (test<int4>(Stride))
+      return 1;
```

Comment:
NIT: this is equivalent to the shorter
```suggestion
    if (test<int2>(Stride) ||
        test<int4>(Stride))
      return 1;
```

Reply:
Thanks, I've applied the suggestion generally.